### PR TITLE
Move the attendance app in

### PR DESCRIPTION
**Pull request by dspencer001**

### CODE REVIEW

File: client/js/attendance/components/home.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+import React                    from 'react';
+import assets                   from '../../libs/assets';
+
+class Home extends React.Component {
```
Comment (BenHSmith13) [inline client/js/attendance/components/home.jsx]: Is this being used? probably delete

File: client/js/attendance/components/layout/index.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+import React                    from "react";
+import { connect }              from "react-redux";
+
+@connect((state) => (state), null, null, {withRef: true})
+class Index extends React.Component {
+
+  constructor(){
```
Comment (BenHSmith13) [inline client/js/attendance/components/layout/index.jsx]: unnecessary constructor

File: client/js/attendance/components/layout/index.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+import React                    from "react";
+import { connect }              from "react-redux";
+
+@connect((state) => (state), null, null, {withRef: true})
```
Comment (BenHSmith13) [inline client/js/attendance/components/layout/index.jsx]: i don't think this needs to be connected

File: client/js/attendance/components/layout/index.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  render(){
+    return<div>
+      {this.props.children}
+    </div>;
+  }
+
+}
+
+export default Index;
```
Comment (BenHSmith13) [inline client/js/attendance/components/layout/index.jsx]: newLine

**Comment by BenHSmith13** [comment]
I'm going to stop before I get too far. Are we just merging adding that other app in and calling it done, or do we want to spend the time doing some TLC on it?

**Comment by dspencer001** [comment]
I was assuming that we would merge it in, and later give it some tlc. Some
code changed from the original, but it was mostly a single action on the
JavaScript side, and the attendances_controller of the rails side. There
were some structure changes though: the apps are now separated out into two
different folders inside the js folder, so all scorm code lives in js/scorm
and all attendance code lives in js/attendance. There are some duplicate
files that could be extracted out(middleware/api.js are in both I think),
but I duplicated them in case we wanted to customize them for each app.
Additionally all styles are separated out too, as the apps have different
styles for the same classes.

**Comment by dspencer001** [comment]
@jbasdf what do you want done with this? And does tagging even work when
responding via email?

**Comment by BenHSmith13** [comment]
This looks good for now. Most of this is being brought form another project and should be refactored, but that may be out of the scope of this PR.

LGTM

**Pull request merged**